api/app/signals/apps/api/v1/views/signal.py

```diff
@@ -117,3 +129,11 @@ def history(self, request, pk=None):
 
         serializer = HistoryHalSerializer(history_entries, many=True)
         return Response(serializer.data)
+
+    @action(detail=False)
+    def geography(self, request):
+        # Makes use of the optimised queryset
+        filtered_qs = self.filter_queryset(self.geography_queryset)
```

Comment:
And here we use the geography_queryset the same way as we should use the "normal" queryset